packages/google_sign_in/ios/Classes/GoogleSignInPlugin.m

```diff
@@ -56,6 +54,8 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [[GGLContext sharedInstance] configureWithError:&error];
     [GIDSignIn sharedInstance].scopes = call.arguments[@"scopes"];
     [GIDSignIn sharedInstance].hostedDomain = call.arguments[@"hostedDomain"];
+    UIViewController *viewController = [UIApplication sharedApplication].keyWindow.rootViewController;
+    [GIDSignIn sharedInstance].uiDelegate = viewController;
```

Comment:
Discussed off-PR - looks like the documentation is wrong, and what we're doing is right